.travis.yml

```diff
@@ -1,8 +1,19 @@
 language: go
 sudo: false
 
-go:
-  - 1.3
-  - 1.4
-  - 1.5
-  - tip
+matrix:
+  include:
+    - go: 1.3
+    - go: 1.4
+    - go: 1.5
+    - go: 1.6
+    - go: tip
```

Comment:
Matrix builds can run in parallel.
On Sat, Feb 27, 2016 at 1:32 AM Dmitri Shuralyov <notifications@github.com>
wrote:

> In .travis.yml
> <https://github.com/gorilla/context/pull/28#discussion_r54331182>:
>
> > @@ -1,8 +1,19 @@
> >  language: go
> >  sudo: false
> >
> > -go:
> > -  - 1.3
> > -  - 1.4
> > -  - 1.5
> > -  - tip
> > +matrix:
> > +  include:
> > +    - go: 1.3
> > +    - go: 1.4
> > +    - go: 1.5
> > +    - go: 1.6
> > +    - go: tip
>
> What's the difference between these two syntaxes? Why use one over the
> other?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/gorilla/context/pull/28/files#r54331182>.
>